spec/models/validations/financial_validations_spec.rb

```diff
@@ -919,39 +919,39 @@
           record.chcharge = 1001
           financial_validator.validate_care_home_charges(record)
           expect(record.errors["chcharge"])
-            .to include(match I18n.t("validations.financial.carehome.out_of_range", min_chcharge: 10, period: "weekly for 52 weeks", max_chcharge: 1000))
+            .to include("Household rent and other charges must be between 10.00 and 1000.00 if paying weekly for 52 weeks")
```

Comment:
I think all these messages should have £ signs prefixing all the financial values